wagon.py

```diff
@@ -1011,6 +1011,94 @@ def repair(source, validate_archive=False):
     return archive_path
 
 
+def _combine_wheels(processed_source_a, processed_source_b, metadata_a,
+                    metadata_b):
+    """ The workdir of the new wagon is processed_source_a dir  """
+    wheels_path_a = os.path.join(processed_source_a, DEFAULT_WHEELS_PATH)
+    wheels_path_b = os.path.join(processed_source_b, DEFAULT_WHEELS_PATH)
+
+    for wheel in _get_downloaded_wheels(wheels_path_b):
+        if not os.path.isfile(os.path.join(wheels_path_a, wheel)):
```

Comment:
Is there a case where the file exists?

Reply:
Sure. if the wheel is py2py3 and both of the wagons has is. Just save time and not copy it.